packages/devtools_app/lib/src/inspector/inspector_controller.dart

```diff
@@ -108,7 +108,7 @@ class InspectorController implements InspectorServiceClient {
 
   final bool isSummaryTree;
 
-  final VoidFunction onExpandCollapseSupported;
```

Comment:
VoidFunction is only a couple chars shorter but I think reads better and is really standard in Flutter.

Reply:
It's 3 characters shorter but doesn't describe the lack of arguments.

It's only standard because the nicer syntax didn't exist when `VoidFunction` was added.

Reply:
> is really standard in Flutter.

It's not used at all in Flutter. I can see an argument that `VoidCallback` is standard, but the fact that both exist with different names and the same meaning is a signal that this is noise. 

The function type is unambiguous and canonical.

The argument that the space makes it hard to read falls apart a bit when we have `VoidFunction` followed immediately by `void Function(dynamic)` in usage...